docs/index.html

```diff
@@ -32,9 +32,9 @@ <h1><code>wrangler</code></h1>
           <h2>🤠 wrangle your cloudflare workers</h2>
         </div>
         <div class="five columns" id="installer">
-          <a class="button button-primary" href="/wrangler/installer">✨ Install wrangler 0.1.1 ✨</a>
-          <p>4 Apr 2019 |
-            <a href="https://github.com/cloudflare/wrangler/releases/tag/v0.1.1">
+          <a class="button button-primary" href="/wrangler/installer">✨ Install wrangler 1.5.0 ✨</a>
+          <p>25 Oct 2019 |
+            <a href="https://github.com/cloudflare/wrangler/releases/tag/v1.5.0">
```

Comment:
This tag will be correct right? not sure if I should omit the `v`

Reply:
yup!